README.md

```diff
@@ -1,17 +1,252 @@
 # Review Service
 
-This repository contains a simple contract to prompt user to review the current application on Android and iOS.
+This repository contains simple service that adds business logic (conditions and data tracking) around a basic review prompter for Android and iOS.
```

Comment:
I would change that to 2 sentences:
This repository introduces abstractions around native review capabilities to ease code sharing and testability.
It also introduces _business logic_ to quickly configure conditions and state tracking to prompt for reviews at the right moment.